daemon/sailfish-secretsd.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Sailfish Secrets Daemon
+Requires=dbus.socket
+After=local-fs.target dbus.socket init-done.service
+Conflicts=shutdown.target
+
+[Service]
+Type=notify
```

Comment:
Why type notify instead of dbus? This service registers entry to dbus, so why don't we use that? Also we should have dbus unit file to bind the dbus service for this systemd service.

The thing here is that if we do not bind the systemd service and dbus unit, things can start this service through dbus eventhough the status would be conflicting e.g. we are heading towards shutdown.target. Which has lead to issues in the past. With binding if someone would try to start this service over dbus it would be prevented by systemd as it maps the same rules than in normal systemd start to dbus start.

Example of how binding can be done.
https://github.com/nemomobile/ssu/pull/35/files